crm_department/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM with Department Categorization
+==================================
+
+
+
+Add the department on Sales Team (with related field on Lead &amp; Opportunities)
+as well as the related filter and button in the search form.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/{project_repo}/issues/new?body=module:%20{module_name}%0Aversion:%20{version}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+*(in alphabetical order)*
+
+* Alexandre Fayolle <alexandre.fayolle@camptocamp.com>
+* Ángel Moya <angel.moya@domatix.com>
+* Charbel Jacquin <charbel.jacquin@camptocamp.com>
+* Daniel Reis <dgreis@sapo.pt>
+* Guewen Baconnier <guewen.baconnier@camptocamp.com>
```

Comment:
@guewen should be remove from there as his contribution is this one:
https://github.com/OCA/department/commit/0ad66c4407f706755de5b1e48f351ddb7c5c155c#diff-ca3d2b4f6d0f160d77cf50818077971a

(I wouldn't personally have my name in all modules I have added some tiny fixed not even knowing the module)